lte/gateway/python/integ_tests/common/gx_gy_client.py

```diff
@@ -0,0 +1,69 @@
+"""
```

Comment:
I do not see this file used anywhere - what am I missing?

Reply:
OK, seems to have been added to s1ap_wrapper in the meantime.

Reply:
I renamed the file and ameded the changes.